ReSwiftTests/StoreTests.swift

```diff
@@ -7,239 +7,262 @@
 //
 
 import XCTest
-import Quick
-import Nimble
 @testable import ReSwift
 
-// swiftlint:disable function_body_length
-// swiftlint:disable type_body_length
-class StoreSpecs: QuickSpec {
+// swiftlint:disable file_length
+class StoreTests: XCTestCase {
 
-    override func spec() {
+    func testInit() {
+        // Dispatches an Init action when it doesn't receive an initial state
 
```

Comment:
What would you think about moving these test comments above the test function? E.g.:

```
// Dispatches an Init action when it doesn't receive an initial state
func testInit() {
   ...
}
```

I file like that might make it more obvious that this is a spec-style comment and a comment related to the source code within the test?